src/stories/index.js

```diff
@@ -0,0 +1,24 @@
+/* global document */
+// better to load this only if it's needed
+import 'document-register-element'; // ES2015
+// load this for browsers which support customElements without builtin (webkit)
+import '@ungap/custom-elements-builtin';
+import { storiesOf } from '@storybook/html';
+
+import globalStyles from '../index.scss';
+
+import './tables';
```

Comment:
@LucaMele 
I think we can fix our grouping issue very very easily with storybook 👍